script/releases/v1.2.0-slashing/6-script/script.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"context"
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"strconv"
+
+	"github.com/attestantio/go-eth2-client/api"
+	attestantio "github.com/attestantio/go-eth2-client/http"
+)
+
+type SignedBeaconBlockResponse struct {
+	Data struct {
+		Message struct {
+			Body struct {
+				ExecutionPayload struct {
+					Timestamp string `json:"timestamp"`
+				} `json:"execution_payload"`
+			} `json:"body"`
+		} `json:"message"`
+	} `json:"data"`
+}
+
+type TArgs struct {
+	BeaconNode string
+	ForkSlot   uint64
+}
+
+func main() {
+	forkSlot, _ := strconv.ParseUint(os.Getenv("ZEUS_ENV_PECTRA_FORK_SLOT"), 10, 64)
+
+	err := runScript(TArgs{
+		BeaconNode: os.Getenv("BEACON_URL"),
+		ForkSlot:   forkSlot,
+	})
+	if err != nil {
+		fmt.Printf("Error: %v\n", err)
+		os.Exit(1)
+	}
+}
+
+func panicOnError(msg string, err error) {
+	if err != nil {
+		fmt.Printf("Error: %s", msg)
+		panic(err)
+	}
+}
+
+func runScript(args TArgs) error {
+	ctx := context.Background()
+
+	beaconClient, err := attestantio.New(ctx,
+		attestantio.WithAddress(args.BeaconNode),
+	)
+	panicOnError("failed to reach beacon node", err)
+
+	httpClient := beaconClient.(*attestantio.Service)
+	// Start checking from fork slot
+	slotNum := args.ForkSlot
+
+	for {
+		opts := &api.BeaconBlockHeaderOpts{Block: strconv.FormatUint(slotNum, 10)}
+		_, err := httpClient.BeaconBlockHeader(ctx, opts)
+		if err != nil {
+			fmt.Printf("Slot %d was missed, checking next slot...\n", slotNum)
+			slotNum++
+			continue
+		}
```

Comment:
are there any other error types we should filter for here? i feel like we upgraded our error handling for the slashing mainnet release